### DISCUSSION TITLE
Docker image drawio connected with private GitHub Enterprise

### DISCUSSION THREAD

**Discussion by LorenzoSantino** [jgraph/drawio#4236] [General]
Hi, using a local docker image, is possibile to connect (only) with the github enteprise of my organization?
I'have tried to follow the suggestion in https://github.com/jgraph/drawio/issues/1572, but  when I start the docker image I can only save file in local file system or browser.

Thanks,
Regards

**Comment by davidjgraph** [comment]
We don't know, we've never tested with github enterprise.
